install.sh

```diff
@@ -16,23 +16,24 @@ LOGS_INSTALL_LOG_FILE_PATH="$LOGS_DIR_PATH/$LOGS_INSTALL_LOG_FILE_NAME"
 
 
 {
+    # pip pywayland requires gcc
     if [[ $(sudo apt-get install 2>/dev/null) ]]; then
         PACKAGE_MANAGER="apt"
-        sudo apt-get -y install ibus libevdev2 curl xinput i2c-tools python3-dev python3-virtualenv libxml2-utils
+        sudo apt-get -y install ibus libevdev2 curl xinput i2c-tools python3-dev python3-virtualenv libxml2-utils libxkbcommon-dev gcc
     elif [[ $(sudo pacman -h 2>/dev/null) ]]; then
         PACKAGE_MANAGER="pacman"
         # arch does not have header packages (python3-dev), headers are shipped with base? python package should contains almost latest version python3.*
-        sudo pacman --noconfirm --needed -S ibus libevdev curl xorg-xinput i2c-tools python python-virtualenv libxml2
+        sudo pacman --noconfirm --needed -S ibus libevdev curl xorg-xinput i2c-tools python python-virtualenv libxml2 libxkbcommon gcc
     elif [[ $(sudo dnf help 2>/dev/null) ]]; then
         PACKAGE_MANAGER="dnf"
-        sudo dnf -y install ibus libevdev curl xinput i2c-tools python3-devel python3-virtualenv libxml2
+        sudo dnf -y install ibus libevdev curl xinput i2c-tools python3-devel python3-virtualenv libxml2 libxkbcommon-devel gcc
     elif [[ $(sudo yum help 2>/dev/null) ]]; then
         PACKAGE_MANAGER="yum"
         # yum was replaced with newer dnf above
-        sudo yum --y install ibus libevdev curl xinput i2c-tools python3-devel python3-virtualenv libxml2
+        sudo yum --y install ibus libevdev curl xinput i2c-tools python3-devel python3-virtualenv libxml2 libxkbcommon-devel gcc
     elif [[ $(sudo zypper help 2>/dev/null) ]]; then
         PACKAGE_MANAGER="zypper"
-        sudo zypper --non-interactive install ibus libevdev2 curl xinput i2c-tools python3-devel python3-virtualenv libxml2
+        sudo zypper --non-interactive install ibus libevdev2 curl xinput i2c-tools python3-devel python3-virtualenv libxml2 libxkbcommon gcc
```

Comment:
```suggestion
        sudo zypper --non-interactive install ibus libevdev2 curl xinput i2c-tools python3-devel python3-virtualenv libxml2 libxkbcommon-devel gcc
```
This one is actually right, I have made a mistake and that's why I resolved the previous conversation. Anyway, I don't know if you know but You can actually apply the suggested changes from the GitHub UI.

Reply:
You can use the commit suggestion button under the suggested change.

Reply:
I know but good point anyway (I was committing something else so I committed that together).

Reply:
Reverted.